ports/aws-c-auth/portfile.cmake

```
@@ -1,10 +1,11 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO awslabs/aws-c-auth
-    REF 61b6524960ad5e0c7aa2e38b343425d5941781bf # v0.6.3
-    SHA512 b5dda92e4a8796f3f1b8e2d326f57979a673f57325c921cdbc9c44273ada2f2a8eb6723f0292d223175ba4cca24508d2b635fad2af5ec7dd9e7b06db9588ede6
+    REF bad1066f0a93f3a7df86c94cc03076fa6b901bd2 # v0.6.22

```

```suggestion
    REF "v${VERSION}"
```
Here could use `v${VERSION}` instead of commit ID, its advantage is avoid next change in next update.